crosschain-resolver/contracts/L1Resolver.sol

```diff
@@ -161,6 +162,9 @@ contract L1Resolver is EVMFetchTarget, ITargetResolver, IMetadataResolver, IExte
             (bytes32 node) = abi.decode(data[4:], (bytes32));
             return _contenthash(node, target);
         }
+        if (selector == IResolverSetter.setAddr.selector) {
```

Comment:
This shouldn't be here - you can't call `setAddr` as part of `resolve`. Instead it needs to be a separate function.